src/staging.c

```diff
@@ -97,13 +97,13 @@ enum swupd_code do_staging(struct file *file, struct manifest *MoM)
 	ret = stat(targetpath, &s);
 	if ((ret == -1) && (errno == ENOENT)) {
 		if (MoM) {
-			warn("Update target directory does not exist: %s. Trying to fix it\n", targetpath);
+			warn("\rUpdate target directory does not exist: %s. Trying to fix it\n", targetpath);
```

Comment:
If we use \r won't the percentages be truncated with the test? Won't we have:

    22% rning: Update ....

And \r are not graceful when printing to a file. journal will probably consider this a blob instead of a text. If it's not bad to use \n I prefer?

Reply:
I have tested this locally and it works as expected. It overwrites the progress percentage and the next value of the percentage is printed t the bottom, as expected. Actually we are already using this same approach for printing output when diagnose find mismatches https://github.com/clearlinux/swupd-client/commit/7d295f3ec5d92f7d0b382a6b3a64026180be5068

Using \n leaves traces of the previous percentage which looks ugly, like this:

```
Warning: Update target directory does not exist: some_dir. Trying to fix....
Path /some_dir is missing on the file system ... fixing
	...39%
Warning: Update target directory does not exist: /usr/share/man/fr/man1. Trying to fix it
Path /usr/share/man/fr is missing on the file system ... fixing
Path /usr/share/man/fr/man1 is missing on the file system ... fixing
	...42%
Warning: Update target directory does not exist: /usr/share/man/man5. Trying to fix it
Path /usr/share/man/man5 is missing on the file system ... fixing
etc...
```

Reply:
That's true. We won't have the problem I said because of the \n at the end of the line.
the only problem now is when output is redirected to files.

Reply:
I submitted an update that I think addresses that problem